src/app/pages/basics/TrafficLayerExample.js

```diff
@@ -0,0 +1,42 @@
+/* global google */
+import {
+  default as React,
```

Comment:
I'll admit, I copied `HeatmapLayer` and changed the relevant bits for `TrafficLayer`. This is in `HeatmapLayer` too. I'm happy update them both if desired, but either way, I'd also love to get `TrafficLayer` into the package.